dbt_project.yml

```diff
@@ -1,10 +1,10 @@
 name: 'twitter_ads_source'
-version: '0.4.0'
+version: '0.5.0'
 config-version: 2
 require-dbt-version: [">=1.0.0", "<2.0.0"]
 models:
   twitter_ads_source:
-    +schema: stg_twitter_ads
+    +schema: twitter_ads_source
```

Comment:
Strangely enough when I run the package **all** my models are `views`. Can we define the default materialization here as well.

Reply:
are `views` the default materialization if you don't specify? i'll make `_tmp` models views and non-tmp ones tables

Reply:
Turns out the default is a [view](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#configuring-materializations). Thanks for updating!